giraffe/src/utils/formatters.ts

```diff
@@ -172,10 +172,15 @@ export const timeFormatter = ({
 
   let formatter
 
+  const getValidDate = (d: number): Date => {
+    const date = new Date(d)
+    return date.getTime() === date.getTime() ? date : new Date()
+  }
+
   if (format) {
     // If a `format` string is passed, we simply use it
     formatter = (x: number) =>
```

Comment:
i know this isn't your code, but any chance we might rename `x` to something that hints at what this supposed to be? even `t` would be better cause it might represent `timestamp`

Reply:
Good call. I have updated it to something more descriptive.